docs/releasing/README.md

```diff
@@ -0,0 +1,384 @@
+    rm vm.bin
+    ```
+
+1. Clone [AvalancheGo](https://github.com/ava-labs/avalanchego):
+
+    ```bash
+    git clone git@github.com:ava-labs/avalanchego.git
+    ```
+
+1. Build AvalancheGo using those VM ids:
+
+    ```bash
+    cd avalanchego
+    ./scripts/build.sh ~/.avalanchego/plugins/mDtV8ES8wRL1j2m6Kvc1qRFAvnpq4kufhueAY1bwbzVhk336o
+    ./scripts/build.sh ~/.avalanchego/plugins/meq3bv7qCMZZ69L8xZRLwyKnWp6chRwyscq8VPtHWignRQVVF
+    ```
+
+1. Get upgrades for each L1 and write them out to `~/.avalanchego/configs/chains/<blockchain-id>/upgrade.json`:
+
+    ```bash
+    mkdir -p ~/.avalanchego/configs/chains/2D8RG4UpSXbPbvPCAWppNJyqTG2i2CAXSkTgmTBBvs7GKNZjsY
+    curl -X POST --silent --header 'Content-Type: application/json' --data '{
+        "jsonrpc": "2.0",
+        "method": "eth_getChainConfig",
+        "params": [],
+        "id": 1
+    }' https://subnets.avax.network/dispatch/testnet/rpc | \
+    jq -r '.result.upgrades' > ~/.avalanchego/configs/chains/2D8RG4UpSXbPbvPCAWppNJyqTG2i2CAXSkTgmTBBvs7GKNZjsY/upgrade.json
+    ```
+
+    Note it is possible there is no upgrades so the upgrade.json might just be `{}`.
+
+    ```bash
+    mkdir -p ~/.avalanchego/configs/chains/98qnjenm7MBd8G2cPZoRvZrgJC33JGSAAKghsQ6eojbLCeRNp
+    curl -X POST --silent --header 'Content-Type: application/json' --data '{
+        "jsonrpc": "2.0",
+        "method": "eth_getChainConfig",
+        "params": [],
+        "id": 1
+    }' https://subnets.avax.network/echo/testnet/rpc | \
+    jq -r '.result.upgrades' > ~/.avalanchego/configs/chains/98qnjenm7MBd8G2cPZoRvZrgJC33JGSAAKghsQ6eojbLCeRNp/upgrade.json
+    ```
+
+1. (Optional) You can tweak the `config.json` for each L1 if you want to test a particular feature for example.
+    - Dispatch: `~/.avalanchego/configs/chains/2D8RG4UpSXbPbvPCAWppNJyqTG2i2CAXSkTgmTBBvs7GKNZjsY/config.json`
+    - Echo: `~/.avalanchego/configs/chains/98qnjenm7MBd8G2cPZoRvZrgJC33JGSAAKghsQ6eojbLCeRNp/config.json`
+1. (Optional) If you want to reboostrap completely the chain, you can remove `~/.avalanchego/chainData/<blockchain-id>/db/pebbledb`, for example:
+    - Dispatch: `rm -r ~/.avalanchego/chainData/2D8RG4UpSXbPbvPCAWppNJyqTG2i2CAXSkTgmTBBvs7GKNZjsY/db/pebbledb`
+    - Echo: `rm -r ~/.avalanchego/chainData/98qnjenm7MBd8G2cPZoRvZrgJC33JGSAAKghsQ6eojbLCeRNp/db/pebbledb`
+
+    AvalancheGo keeps its database in `~/.avalanchego/db/fuji/v1.4.5/*.ldb` which you should not delete.
+1. Build AvalancheGo:
+
+    ```bash
+    ./scripts/build.sh
+    ```
+
+1. Run AvalancheGo tracking the Dispatch and Echo VM IDs:
+
+    ```bash
+    ./build/avalanchego --network-id=fuji --partial-sync-primary-network --public-ip=127.0.0.1 \
+    --track-subnets=7WtoAMPhrmh5KosDUsFL9yTcvw7YSxiKHPpdfs4JsgW47oZT5,i9gFpZQHPLcGfZaQLiwFAStddQD7iTKBpFfurPFJsXm1CkTZK
+    ```
+
+1. Follow the logs and wait until you see line stating the health `check started passing`, for example:
```

Comment:
we should probably also check for `consensus started` logs. (and also bootstrapped healthy nodeS)